.travis.yml

```diff
@@ -121,41 +120,41 @@ install:
       # dependencies
       brew update > /dev/null
       brew outdated "pkg-config" || brew upgrade "pkg-config"
-      brew install colormake ccache zlib qt
+      brew install colormake ccache zlib qt libtorrent-rasterbar
```

Comment:
+1 for that. Or we can build directly from the libtorrent repo, which does not take long.